app/scripts/lib/snap-keyring/metrics.ts

```diff
@@ -0,0 +1,51 @@
+import MetamaskController from '../../metamask-controller';
+
+export default async function getSnapAndHardwareInfoForMetrics(
```

Comment:
We should add unit test for the methods in this file

Reply:
Good idea, I'm going to add some tests in app/scripts/controllers/transactions/index.test.js

I thought about adding a unit test for just this file, but we'd basically just be testing a lot of stubs

Reply:
I think that holds some value since you want to make sure it calls the correct methods and the data is getting passed correctly. Someone could change a return object and the test+linter should be able to catch any potential bug

Reply:
To be done in a follow up PR